### DISCUSSION TITLE
How many fast radio bursts (FRBs) have already been localized so far?

### DISCUSSION THREAD

**Discussion by kongjunZhang** [chime-frb-open-data/community#28] [Q&A] [answered]
Although I looked it up in the database, there is no specific catalog for lacalized FRBs

**Comment by emilypetroff** [comment, marked as answer]
The larger sample of localized FRBs is catalogued on the "FRB Hosts" webpage [http://frbhosts.org/](http://frbhosts.org) . Note that this page is not run by CHIME/FRB but does include localized CHIME/FRB events.

**Comment by kongjunZhang** [comment]
Thank you very much for your selfless help!
